content/en/blog/_posts/2021-10-04-clusterclass-and-managed-topologies.md

```diff
@@ -0,0 +1,143 @@
+
+Fast forwarding to 2021, with lots of companies using Cluster API to manage fleets of Kubernetes clusters running workloads in production, the community focused its effort on stabilization of both code, APIs and documentation, and on extensive test signals which inform Kubernetes releases.
+
+With solid foundations in place, and a vibrant and welcoming community that still continues to grow, it was time to plan another iteration on our UX for both new and advanced users.
+
+Enter ClusterClass and manage topologies, tada!
+
+## ClusterClass
+
+As the name suggests, ClusterClass and managed topologies are built in two parts.
+
+The idea behind ClusterClass is simple: let’s define the shape of your cluster once, and then reuse it many times, abstracting the complexities and the internals of a Kubernetes cluster away.
+
+![Defining a ClusterClass](/images/blog/2021-10-04-clusterclass-and-managed-topologies/clusterclass.svg)
+
+ClusterClass, at its heart, is a collection of Cluster and Machine templates. You can use it as a “stamp” that can be leveraged to create many clusters of a similar shape.
+
+```yaml
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+ kind: ClusterClass
+ metadata:
+   name: my-amazing-cluster-class
+   namespace: bar
+ spec:
+   controlPlane:
+     ref:
+       apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+       kind: KubeadmControlPlaneTemplate
+       name: high-availability-kcp
+     machineInfrastructure:
+       ref:
+         apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+         kind: VSphereMachineTemplate
+         name: controlplane-vsphere-machinetemplate
+   workers:
+     deployments:
+     - class: linux-worker
+       template:
+         bootstrap:
+           ref:
+             apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+             kind: KubeadmConfigTemplate
+             name: linux-bootstrap
+         infrastructure:
+           ref:
+             apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+             kind: VSphereMachineTemplate
+             name: linux-vsphere-template
+     - class: windows-worker
+       template:
+         bootstrap:
+           ref:
+             apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+             kind: KubeadmConfigTemplate
+             name: windows-bootstrap
+         infrastructure:
+           ref:
+             apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+             kind: VSphereMachineTemplate
+             name: windows-vsphere-template
+   infrastructure:
+     ref:
+       apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
+       kind: VSphereClusterTemplate
+       name: vsphere-cluster
+```
+
+The possibilities are endless; you can get a default ClusterClass from the community, “off-the shelf” classes from your vendor of choice, “certified” classes from the platform admin in your company, or even create custom ones for advanced scenarios.
```

Comment:
seems like a couple dictionaries suggest that "off-the-shelf" should have hyphens between all words
```suggestion
The possibilities are endless; you can get a default ClusterClass from the community, “off-the-shelf” classes from your vendor of choice, “certified” classes from the platform admin in your company, or even create custom ones for advanced scenarios.
```